src/v/compat/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.12.0)
+
+if (NOT EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/bin)
```

Comment:
please move this to oss.cmake like so

 https://github.com/redpanda-data/redpanda/blob/dev/cmake/oss.cmake.in#L119-L150

there are custom `INSTALL` steps and it is guaranteed to run once and works w/ the cmake caching strategies.